src/plugins/tagfilter/tagfilter.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * @title WET-BOEW Tag filter
+ * @overview Filter based content tagging
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, document, wb ) {
+"use strict";
+
+var componentName = "wb-tagfilter",
+	selector = ".provisional." + componentName,
+	controllerName = componentName + "-ctrl",
+	selectorCtrl = "." + controllerName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	filterOutClass = "wb-tgfltr-out",
+	filterInClass = "wb-tgfltr-in",
+	filterExclusiveClass = "wb-tgfltr-exclusive",
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+
+		if ( elm ) {
+			$elm = $( elm );
+
+			// Find the controllers, this element must have an id. It assumes the controller only controls one tag-filtering section
+			var controllers = $.find( "[aria-controls=" + elm.id + "]" );
+
+			if ( controllers.length === 0 ) {
+				console.warn( "Need to add a default controller" );
+			} else {
+				$( controllers ).addClass( controllerName ); // Add a class for event binding.
+			}
+
+			wb.ready( $elm, componentName );
+		}
+	};
+
+// Add or Remove filter when the checkbox is selected
+$document.on( "change", "input:checkbox" + selectorCtrl, function( event )  {
+
+	var elm = event.currentTarget,
+		filterTag = elm.value,
+		state = !!elm.checked,
+		controlsId = elm.getAttribute( "aria-controls" ),
+		relatedPotential = document.querySelectorAll( "#" + controlsId + " [data-wb5-tags*=" + filterTag + "]" ),
+		related = [],
+		relatedExclusive = [],
+		relatedNot = [],
+		relatedExclusiveRem = [],
+		i, i_len, j, j_len,
+		currentElm, tagList, tag, lastIndex;
+
+	// Filter down the ones that matched the initial DOM search
+	i_len = relatedPotential.length;
+	for ( i = 0; i < i_len; i = i + 1 ) {
+		currentElm = relatedPotential[ i ];
+
+		tagList = currentElm.dataset.wb5Tags.split( " " );
```

Comment:
@Garneauma Please call it `wbTags`:
```suggestion
		tagList = currentElm.dataset.wbTags.split( " " );
```